<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PackageManager</class>
 <widget class="QMainWindow" name="PackageManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Package Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTableWidget" name="tableWidget">
       <property name="columnCount">
        <number>3</number>
       </property>
       <column/>
       <column/>
       <column/>
      </widget>
      <widget class="QTreeView" name="treeView"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionNew_Package"/>
    <addaction name="actionDelete_Selected"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuBuild">
    <property name="title">
     <string>Build</string>
    </property>
    <addaction name="actionBuild_All"/>
    <addaction name="actionBuild_Selected"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuBuild"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionBuild_All"/>
   <addaction name="actionBuild_Selected"/>
   <addaction name="separator"/>
   <addaction name="actionNew_Package"/>
   <addaction name="actionDelete_Selected"/>
  </widget>
  <action name="actionNew_Package">
   <property name="text">
    <string>New Package</string>
   </property>
   <property name="toolTip">
    <string>Create new package</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save changes</string>
   </property>
  </action>
  <action name="actionDelete_Selected">
   <property name="text">
    <string>Delete Selected</string>
   </property>
   <property name="toolTip">
    <string>Delete selected package</string>
   </property>
  </action>
  <action name="actionBuild_Selected">
   <property name="text">
    <string>Build Selected</string>
   </property>
   <property name="toolTip">
    <string>Build selected package</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Close Package Manager</string>
   </property>
  </action>
  <action name="actionBuild_All">
   <property name="text">
    <string>Build All</string>
   </property>
   <property name="toolTip">
    <string>Build all packages</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>PackageManager</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
